scripts/install-chef-suse.sh

```diff
@@ -287,11 +304,17 @@ Aborting: Administration Server is already deployed.
 If you want to run the installation script again,
 then please remove the following file:
 
-    /opt/dell/crowbar_framework/.crowbar-installed-ok
+    $crowbar_install_dir/crowbar-installed-ok
 EOF
     exit 1
 fi
 
+if [ -f $crowbar_install_dir/crowbar-install-failed ] || [ "$CROWBAR_WIZARD_MODE" ]; then
```

Comment:
I asked earlier, but: Why not always do this? I don't see a case where we wouldn't want to do these things.